openedx/features/badging/helpers.py

```diff
@@ -71,6 +71,10 @@ def get_course_badges(user, course_id, earned_badges, badge_queryset=None):
                 # user has not joined any team
                 badges['team_joined'] = False
 
+            if course_team and 'team_id' in course_team and course_team['team_id']:
+                # only if user has joined any team
+                badges['team_id'] = course_team['team_id']
+
```

Comment:
Let's move `team_id` to constants file and import from there

Reply:
Done